### PR TITLE
Hack to fix placeholder issue for Chrome

### PR DESCRIPTION
**Pull request by mrpau-eugene**
### Summary

After some further reading about Chrome's autofill, it seems like it's not a bug on their part but rather some sort of security feature that if Chrome autofill is enabled, it doesn't update the value of the `[input type="password"]` if the user doesn't interact with the page (e.g. clicking anywhere or pressing anything on the keyboard).

Seems like many have tried simulating a users actions by focusing from one element and then focus on another in order to trigger the event that adds value to the password but none of them were successful since Chrome has also fixed this.

This fix was inspired from @indirectlylit's fix (#3232) by disabling the floatingLabel when it is autofilled by chrome. It feels more natural (at least for me). I also increased the timeout since `100ms` doesn't guarantee that chrome's autofill will trigger. Some also claim that `200ms` isn't also a guarantee. `300 to 500` could be an ideal number but it might cause some performance issues. `250ms` for me seems to be a bit good. Although password placeholder still appears (but rarely).

### Reviewer guidance

1. Enable chrome autofill.
1. Try refreshing the page many times.
1. See if placeholder still exists

### References

…

----

### Contributor Checklist

- [ ] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] Contributor has fully tested the PR manually
- [ ] Screenshots of any front-end changes are in the PR description
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

**Comment by mrpau-eugene** [comment]
@radinamatic could you test this and see if placeholder still persists?

**Comment by radinamatic** [comment]
@mrpau-eugene I followed the reviewer guidance: autofill in Advanced settings is enabled by default, but either with leaving it like that, or by disabling it and refreshing several times, and I still see the issue... :disappointed: 

workspace 1_057

If the Chrome devs left this feature so tied in the knots, with barely an option to modify it, maybe we should not invest too much time it trying to fix it...? 
cc @indirectlylit

File: kolibri/core/assets/src/views/k-textbox/index.vue [hunk 1]
```diff
@@ -107,6 +107,13 @@
         type: Boolean,
         default: false,
       },
+      /**
+       * Whether or not to display as a floating label
+       */
+      floatingLabel: {
+        type: Boolean,
+        default: true,
+      },
```
Comment (indirectlylit) [inline kolibri/core/assets/src/views/k-textbox/index.vue:117]: Note that `k-textbox` is a core component in the styleguide, and we need to be judicious about adding new props.

I'm going to make this prop `private` so it doesn't increase the API surface area.

**Comment by indirectlylit** [comment]
well, the change "works for me", so at the very least it mitigates the problem in some cases:

image

merging in!

**Pull request merged**